src/System.Runtime.Serialization.Json/tests/System.Runtime.Serialization.Json.Tests.csproj

```diff
@@ -10,17 +10,23 @@
     <RootNamespace>System.Runtime.Serialization.Json.Tests</RootNamespace>
     <AssemblyName>System.Runtime.Serialization.Json.Tests</AssemblyName>
     <ProjectGuid>{701CB3BC-00DC-435D-BDE4-C5FC29A708A7}</ProjectGuid>
+    <UseProjectReferences>true</UseProjectReferences>
+  </PropertyGroup>
+  <PropertyGroup Condition="!$(UseProjectReferences)" >
```

Comment:
The above PropertyGroup always set the variable to true. To change the value to false we need to change this file. Can we make it possible for us to set the value via environment variable so that we don't have to change the file when running the tests against desktop?